<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>predikcija</class>
 <widget class="QMainWindow" name="predikcija">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>888</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>917</width>
    <height>888</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>917</width>
    <height>888</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Treniranje Deep Learning modela za predikciju mreznog saobracaja</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(91, 91, 91);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="aplikacijeButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Prepoznavanje
aplikacija
u fajlu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ucitavanjeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Odabir fajla</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>701</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 91, 91);
</string>
    </property>
    <property name="title">
     <string>Putanja fajla:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>20</width>
      <height>751</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="selektujButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>850</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Selektuj sve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deselektujButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>850</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deselektuj sve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="treningButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>850</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokreni trening modela</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>901</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(91, 91, 91);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>831</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(91, 91, 91);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>130</y>
      <width>331</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="aplikacijeLayout"/>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>551</width>
      <height>751</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>aplikacijeButton</zorder>
   <zorder>ucitavanjeButton</zorder>
   <zorder>groupBox</zorder>
   <zorder>line</zorder>
   <zorder>label</zorder>
   <zorder>selektujButton</zorder>
   <zorder>deselektujButton</zorder>
   <zorder>treningButton</zorder>
   <zorder>frame</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>textEdit</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
